composer.json

```diff
@@ -43,7 +43,7 @@
         "tubalmartin/cssmin": "2.4.8-p4",
         "magento/magento-composer-installer": ">=0.1.11",
         "braintree/braintree_php": "3.7.0",
-        "symfony/console": "~2.3 <2.7",
```

Comment:
In order to avoid issue with depreciation exceptions from Symfony code I'd recommend change this dependency in the following way:
```
"symfony/console": "~2.3, !=2.7.0",
```

This issue was introduced in 2.7.0 and was fixed in [2.7.1](https://symfony.com/blog/symfony-2-7-1-released)
```
bug #14900 Silence deprecation warnings by default (reecefowell)
```